server/command.go

```diff
@@ -278,6 +283,55 @@ func executeHelp(p *Plugin, c *plugin.Context, header *model.CommandArgs, args .
 	return responsef(helpText)
 }
 
+func executeImport(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
```

Comment:
0/5 We may want to support 2 kinds of "import": **replace** which is what you have here, and **update** when the new config is applied as an update, matching on Name (Pattern as a secondary?). The matched entries are updated, the unmatched ones are added?

cc @lieut-data for an opinion - I think he might advocate sticking with the idempotent **replace** only, as is. For "manual" use by a sysadmin, having an update makes a lot of sense.

One reason to have the "update" mode would be that we can then have library of snippets in this repo, that customers can customize and upload one at a time. Can add a link to the plugin's config page :)

Reply:
Would the update functionality be implemented as a separate function (i.e. `executeUpdate`), or as a flag that gets checked in the `executeImport` function (i.e. `if isUpdate == true {...}`)?

Reply:
after you said it, I think 2 separate commands, `/autolink replace` and `/autolink update` would make most sense.